components/script/dom/defaultteereadrequest.rs

```diff
@@ -31,9 +32,13 @@ pub(crate) struct DefaultTeeReadRequestMicrotask {
     tee_read_request: Dom<DefaultTeeReadRequest>,
 }
 
-impl DefaultTeeReadRequestMicrotask {
-    pub(crate) fn microtask_chunk_steps(&self, cx: SafeJSContext, can_gc: CanGc) {
-        self.tee_read_request.chunk_steps(cx, &self.chunk, can_gc)
+impl MicrotaskRunnable for DefaultTeeReadRequestMicrotask {
+    fn handler(&self, cx: SafeJSContext, can_gc: CanGc) {
+        self.tee_read_request.chunk_steps(cx, &self.chunk, can_gc);
```

Comment:
We can use `GlobalScope::get_cx()` instead of passing the context as an argument.